src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonConverterOfT.cs

```diff
@@ -189,6 +189,7 @@ internal bool TryRead(ref Utf8JsonReader reader, Type typeToConvert, JsonSeriali
                 }
 
                 value = default;
+                populatedValue = default;
```

Comment:
Why not just `false`?

Reply:
I used `default` here as replacement for `I don't care because we return false anyway`